orangecontrib/spectroscopy/widgets/owfft.py

```diff
@@ -401,7 +402,31 @@ def calculateFFT(self):
                 phase_corr=self.phase_corr,
                 peak_search=self.peak_search,
                 )
-
+                
+        if self.reader == 'NeaReaderGSF':
+            fft_single = irfft.ComplexFFT(
+                    dx=self.dx,
+                    apod_func=self.apod_func,
+                    zff=2**self.zff,
+                    phase_res=self.phase_resolution if self.phase_res_limit else None,
+                    phase_corr=self.phase_corr,
+                    peak_search=self.peak_search,
+                    )
+            full_data = self.data.X[::2] * np.exp(self.data.X[1::2]* 1j)
+            for row in full_data:
+                spectrum_out, phase_out, wavenumbers = fft_single(
+                    row, zpd=stored_zpd_fwd, phase=None)
```

Comment:
You can leave out `phase`, it defaults to None and you don't use it.

Reply:
I thought it was doing something, but I changed it and nothing happened, so I deleted that command